src/main/java/frc/robot/RobotContainer.java

```diff
@@ -83,15 +84,14 @@ public RobotContainer() {
      * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
      */
     private void configureButtonBindings() {
-        UserControls controls = new XboxUserControls(0, 1);
+        UserControls controls = new MixedXboxJoystickControls(0, 1, 5);
 
         // Define commands here
         ArcadeDrive defaultDriveCommand = new ArcadeDrive(drive,
                 () -> controls.getLeftDriveX(),
                 () -> controls.getLeftDriveY(),
                 () -> controls.getRightDriveX(),
```

Comment:
I would suggest keeping sniper mode and just make it never called bc I think Matthew implied that he would switch back to xbox if we got working xbox controllers

Reply:
It seems Matthew would rather forfeit the match than stoop as low as using sniper mode.

Reply:
ok